src/util/EntityToFormMapper.js

```diff
@@ -248,17 +276,22 @@ const generateFormSchemaField = (attribute, mapperOptions?: MapperOptions): Form
   // options is a function that always returns an array of option objects
   const options = getFieldOptions(attribute, mapperOptions)
   let fieldProperties = {
-    type: getHtmlFieldType(attribute.fieldType),
     id: attribute.name,
     label: attribute.label,
     description: attribute.description,
+    type: getHtmlFieldType(attribute.fieldType),
     required: isRequired(attribute),
     disabled: attribute.readOnly || attribute.fieldType === 'ONE_TO_MANY',
```

Comment:
in the "old" forms the one to many is not disabled and in some magic way, it is also functional

Reply:
Have to look in to this, i do not know why this is , i did not change it in the PR, suggest to leave it as is for now.